### PR TITLE
Remove volume labels use

### LINKED ISSUE #507: Remove volume labels use

### PR DESCRIPTION
**Pull request by ezodude**
Resolves #507 

- Removes the label from Volume Config.
- Removes volume labels from test fixtures.

### CODE REVIEW

File: pkg/kev/config/labels.go [hunk 1]
```diff
@@ -53,17 +53,13 @@ const (
 	LabelServiceExposeTLSSecret = LabelPrefix + "service.expose.tls-secret"
 
 	// LabelVolumeSize defines persistent volume size
-	LabelVolumeSize = LabelPrefix + "volume.size"
+	// LabelVolumeSize = LabelPrefix + "volume.size"
```
Comment (marcinc) [inline pkg/kev/config/labels.go:56]: Should we drop the volume labels?

File: pkg/kev/kev_reconcile_test.go [hunk 2]
```diff
@@ -114,15 +114,21 @@ var _ = Describe("Reconcile", func() {
 				})
 			})
 
-			When("the override volume label overrides have been updated", func() {
+			When("the override volume extensions overrides have been updated", func() {
```
Comment (marcinc) [inline pkg/kev/kev_reconcile_test.go:117]: small reword suggestion: `the override volume extensions overrides have been updated` -> `the override volume extensions have been updated`.

**Review by marcinc — DISMISSED** [review]
couple of minor comments. LGO. 👍

File: pkg/kev/kev_reconcile_test.go (hunk 2 above)
Comment (ezodude) [reply pkg/kev/kev_reconcile_test.go:117]: Yup that one really read funny.

**Review by marcinc — APPROVED** [review]
👍

**Pull request merged**